components/timeline/spec/Timeline.ts

```diff
@@ -0,0 +1,33 @@
+import { createElement as h } from 'react';
+import { mount } from '@not-govuk/component-test-helpers';
+import Timeline from '../src/Timeline';
+
+describe('Timeline', () => {
+  describe('when given valid props', () => {
```

Comment:
Would be nice to have one with minimal props (no `by`, for example) and one will all props.